extension/firefox/chrome/ShumwayCom.jsm

```diff
@@ -273,6 +291,98 @@ var ShumwayCom = {
           return;
         }
         onSystemResourceCallback = callback;
+      },
+      createLocalConnection: function(connectionName, callback) {
```

Comment:
Move this code into separate jsm-files (similar to RtmpUtils and SpecialInflateUtils).  You can create e.g. `localConnections` object that will encapsulate the logic, the _getLocalConnection and localConnectionsRegistry can be moved there as well.